### DISCUSSION TITLE
Tapo P100 Plug: Error communication with API: Cannot connect to host (Even thought it's set to Static IP)

### DISCUSSION THREAD

**Discussion by Burak-KW** [petretiandrea/home-assistant-tapo-p100#348] [Q&A]
P100 Plug Says: `Error communication with API: Cannot connect to host` when my router has been turned off and on again. 
Even though it is set to a reserved IP. It fixes its self after a day or two but any way to fix this so it doesn't happen?
![image](https://user-images.githubusercontent.com/119974619/213256349-bee99734-5f22-4326-ac12-95ee8f0152c1.png)

**Comment by konbozikis** [comment]
I have been having the same issue lately, ip is static. Usually it can be fixed by unplugging the plug for a while. Haven't found a cause for this
